gridspeccer/core.py

```diff
@@ -181,18 +181,31 @@ def plot_labels(
     label_zpos=None,
     label_color=None,
     fontdict=None,
+    latexformat=True,
+    listoflabels=string.ascii_lowercase,
 ):
-    "plot labels"
+    """plot labels
+
+    Parameters
+    ----------
+    latexformat : bool
+        depending on whether true or fake latex is used, the formatting calls
+        have to be included or left out
+    listoflabels: iterable
+        List of labels to be used for labelling. by default lowercase letters,
+        but can be any list to accomodate uppercase or b1, b2 like labels.
+    """
     label_xpos = label_xpos if label_xpos is not None else {}
     label_ypos = label_ypos if label_ypos is not None else {}
     label_zpos = label_zpos if label_zpos is not None else {}
     label_color = label_color if label_color is not None else {}
 
-    for label_idx, char in zip(labels_to_plot, string.ascii_lowercase):
+    for label_idx, char in zip(labels_to_plot,
+                               listoflabels):
         log.info("Subplot %s receives label %s", label_idx, char)
         plot_caption(
             axes[label_idx],
-            "\\textbf{" + char + "}",
+            "\\textbf{" + char + "}" if latexformat else char,
```

Comment:
Suggestion: Have `latexformat` be a formatstring defaulting to `\\latex{{{}}}` (braces have to escaped by doubling) so that this line could be `latexformat.format(char)`, thereby also allowing for `\\textit` etc.

Reply:
:+1:  implemented

Reply:
One could parse `mpl.rcParams['text.usetex']` to automatically remove latex formatting, but I guess that would be too much magic and it is better to do it in the 'user' scripts.